README.md

```diff
@@ -126,6 +126,40 @@ const output: PluginOutput = {
 };
 ```
 
+## Plugin Quick Start
+
+The kernel supports 2 types of plugins:
+1. Github actions ([wiki](https://github.com/ubiquity/ubiquibot-kernel/wiki/How-it-works))
```

Comment:
Where does this point to?

Reply:
It points to the https://github.com/ubiquity/ubiquibot-kernel/wiki/How-it-works page which describes how the kernel works with "github action style" plugins.